.gitignore

```diff
@@ -4,6 +4,8 @@
 temp/
 build/
 dist/
+.idea/
```

Comment:
These two lines should not be committed. They should be part of either your global gitignore (`core.excludesFile`), or your project's local exclude file (`$GIT_DIR/info/exclude`). See [gitignore docs][1] for details.

[1]: https://git-scm.com/docs/gitignore#_description

Reply:
Done